cmd/k8s-bigip-ctlr/main.go

```diff
@@ -102,14 +104,15 @@ var (
 	// package variables
 	isNodePort         bool
 	watchAllNamespaces bool
+	VXLAN_NAME         string
```

Comment:
What's the difference between VXLAN_NAME and openshiftSDNName above? Also, why is it not camelCase?

Reply:
I can change it to be camelCase. This variable is used to represent either the flannel tunnel name or the openshift sdn name, for simplicity's sake later on in the code.